### PR TITLE
[WIP] macos arm64 support

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@conda-forge/r, @conda-forge/core, @mingwandroid, what do you think about the changes for cross compiling?

**Comment by xhochy** [comment]
Neat, from the diff it just looks like the same thing we're doing in the Python recipes?

**Comment by mingwandroid** [comment]
Fine by me!

**Review by ocefpaf — APPROVED** [review]

File: recipe/build.sh [hunk 1]
```diff
@@ -1,36 +1,4 @@
...
-            install_name_tool -change /usr/lib/libexpat.1.dylib "${PREFIX}"/lib/libexpat.1.dylib "${SHARED_LIB}" || true
-            install_name_tool -change /usr/lib/libcurl.4.dylib "${PREFIX}"/lib/libcurl.4.dylib "${SHARED_LIB}" || true
-            install_name_tool -change /usr/lib/libc++.1.dylib "${PREFIX}"/lib/libc++.1.dylib "${SHARED_LIB}" || true
-            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libc++.1.dylib "${PREFIX}"/lib/libc++.1.dylib "${SHARED_LIB}" || true
-          done <   <(find . \( -type f -iname "*.dylib" -or -iname "*.so" -or -iname "R" \) -print0)
-        popd
-      done
-    popd
-  fi
-fi
```
Comment (ocefpaf) [inline recipe/build.sh:36]: This comes from the skeleton. If we are removing it from the recipe we should probably upstream that. Not sure if it is conda-build skeleton or the post script that the r-team runs.

Comment (isuruf) [reply recipe/build.sh:36]: Fixed in https://github.com/conda/conda-build/pull/4143

Comment (ocefpaf) [reply recipe/build.sh:36]: As usual you are a few steps ahead of me :smile:

**Comment by isuruf** [comment]
`R_ARGS` addition is upstreamed in https://github.com/conda/conda-build/pull/4144
I don't have a PR to add `cross-r-base` upstream. I can send a PR.

File: recipe/build.sh (hunk 1 above)
Comment (jdblischak) [reply recipe/build.sh:36]: I'm very happy to finally get rid of all that unused logic in the build script. We used to try and remove all the extra unnecessary code in `build.sh` with the conda-forge helper script, but it eventually got so complicated, and it wasn't hurting anything but the readability, that we stopped trying to remove the extra code added by newer versions of the skeleton.

**Pull request closed without merging**